src/assets/text/how-to-play.ts

```diff
@@ -46,22 +45,21 @@ export const howToPlay = {
     },
     {
       regular: "No messages sent or no selection submitted: ",
-      highlight: "Results invalidated.",
+      highlight: "0 points.",
     },
     {
       regular:
         "Hidden achievements grant additional points. You'll have to keep playing to find them all.",
       highlight: "",
     },
   ],
-  dailyLeaderboard: [
-    "Ranks are based on daily total scores.",
-    "Aleo credits are awarded to top 10 players automatically when the leaderboard resets.",
-    "Leaderboard resets daily at 00:00 PST.",
+  leaderboard: [
+    "Ranks are based on total scores.",
+    "Scores in each round make up a players total score",
   ],
   playerProfiles: [
     {
-      regular: "Anon players are ineligible for leaderboard/credits.",
+      regular: "Anon players are ineligible for leaderboard position.",
```

Comment:
```suggestion
      regular: "Anonymous players are ineligible for leaderboard position.",
```
idk "Anon" doesn't seem straightforward if you are not a designer or developer

Reply:
too close to Qanon 😬